src/components/application_manager/src/hmi_state.cc

```diff
@@ -323,6 +331,10 @@ mobile_apis::VideoStreamingState::eType AudioSource::video_streaming_state()
 mobile_apis::HMILevel::eType AudioSource::max_hmi_level() const {
   using namespace mobile_apis;
 
+  if (parent() && HMILevel::INVALID_ENUM == parent()->hmi_level()) {
+    return parent()->hmi_level();
+  }
```

Comment:
```suggestion
  if (HMILevel::INVALID_ENUM == parent_hmi_level()) {
    return parent_hmi_level();
  }
```
This function has a null check for `parent()` when necessary